dandischema/datacite.py

```diff
@@ -176,7 +176,7 @@ def to_datacite(
         attributes["relatedIdentifiers"] = []
         for rel_el in meta.relatedResource:
             if rel_el.identifier is not None:
-                if rel_el.identifier.lower().startswith("doi"):
+                if rel_el.identifier.lower().startswith("doi:"):
```

Comment:
since `rel_el.identifier` starts with http, won't this require an example that starts with `doi:`

Reply:
so, this part is actually meant to cover examples like this:https://github.com/dandi/dandischema/blob/master/dandischema/tests/test_datacite.py#L257
but before my changes it was also catching urls without http

Reply:
commented because codecov said it wasn't covered. it's possible that it missed it because of some other failure.

Reply:
the tests cover the check and there tests that go to the `if` part and to the `else` part